### ISSUE TITLE
Make shares on twitter / yammer / other places look nice (by adding some metadata)

### ISSUE THREAD

**Issue by andylolz** [IATI/IATI-Standard-Website#192]
Obviously this is just a nice-to-have, and by the looks of things aidtransparency.net had the same issue so it’s not new.

---

It would be cool to add `title`, `description` and open graph & twitter tags such as `og:image`, plus `keywords` and other stuff for discoverability.

Looks like this right now:

```html
    <head>
        …
        <title>
            
                Home
            
            
                
                    
                
            
        </title>
        <meta name="description" content="" />
        …
    </head>
```

So shares look like this:

![screen shot 2018-07-08 at 19 45 27](https://user-images.githubusercontent.com/464193/42422899-8e00d29a-82e7-11e8-992d-ba6372991ee5.png)

**Comment by andylolz** [comment]
I just bumped into this again, when posting a news article on yammer:

![screen shot 2018-08-24 at 10 35 36](https://user-images.githubusercontent.com/464193/44577620-7def6800-a789-11e8-81d0-eeb8df0e0929.png)

A description and default image would be cool here.

**Comment by andylolz** [comment]
Seems like this was fixed in #336. Cool!

**Comment by samuele-mattiuzzo** [comment]
cool!
